cms/templatetags/cms_js_tags.py

```diff
@@ -48,12 +48,14 @@ def _render_plugin_js(plugin):
         try:
             child_classes = plugin_children[plugin.plugin_type]
         except KeyError:
-            child_classes = plugin.get_plugin_class().get_child_classes(slot=slot, page=page, instance=plugin)
+            plugin_class = plugin_pool.plugins[plugin.plugin_type]
```

Comment:
Its ok, adding the method to the class won't help us here as I still want to avoid calling method + pool for each plugin.